### ISSUE TITLE
[Catalog] ActivityIndicator example does not handle dynamic type correctly

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#3686]
### Overview
The ActivityIndicator main "Demo" has visual bugs when larger text sizes are used.

![simulator screen shot - iphone x - 2018-05-03 at 11 43 28](https://user-images.githubusercontent.com/1753199/39587355-4f5f222a-4ec7-11e8-9c8b-f0b64beb49c0.png)

### Reproduction steps
1. Turn on larger text sizes in the Settings App &rarr; General &rarr; Larger Text
1. Launch the catalog
1. Tap on Activity Indicator &rarr; Start Demo

### Version number
55.0.0

### Operating system and device
iPhone X/iOS 11.3 (Simulator)

---
#### Internal data
- Associated internal bug: [b/117179107](http://b/117179107)

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
